docs/organizations/using-a-coding-standard.md

```diff
@@ -5,24 +5,27 @@
 
     For more information [read the release announcement](https://blog.codacy.com/organization-coding-standards/) or [watch a demo (3 min)](https://www.loom.com/share/19642d09662e40f2820bf2be6bdf3660) to learn how to create a coding standard for your organization.
 
-Create a coding standard on your organization to define and apply shared tool and code pattern configurations consistently across your repositories. You can also apply the coding standard to new repositories automatically by defining the coding standard as default.
+Create coding standards on your organization to define and apply shared tool and code pattern configurations consistently across your repositories. You can also apply a coding standard to new repositories automatically by defining it as default.
 
-The organization coding standard helps you ensure that Codacy analyzes multiple repositories with the same tool and code pattern settings. For example, you can use the organization coding standard to ensure that all your repositories and teams are following the same security rules or coding conventions.
+Coding standards help you ensure that Codacy analyzes multiple repositories with the same tool and code pattern configurations. For example, you can use a coding standard to ensure that a team or a group of repositories follow the same security rules or coding conventions.
```

Comment:
I would be more specific in saying that the multiple repositories that Codacy analyses are the ones you selected:

`Coding standards help you ensure that Codacy analyzes the multiple repositories you selected / associated to a coding standard with the...`

Reply:
In my opinion this adds some extra complexity to the sentence with no extra clarity about how the feature works, as it's already implicit that only some repositories (the ones you select) are affected by each coding standard.

Reply:
I second Paulo's suggestion. Resolving this conversation